includes/amp-helper-functions.php

```diff
@@ -532,9 +533,21 @@ function _amp_bootstrap_customizer() {
  *
  * @since 0.7
  *
+ * @param bool $ignore_late_defined_slug Whether to ignore the late defined slug.
```

Comment:
The signature change should warrant a new since annotation.
```suggestion
 * @since 0.7
 * @since 2.1 Added $ignore_late_defined_slug argument.
 *
 * @param bool $ignore_late_defined_slug Whether to ignore the late defined slug.
```